Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.10
+WORKDIR /backup
+
+ENV ACCESS_TOKEN=""
+ENV ORGANIZATION=""
+
+RUN apt-get update && apt-get upgrade -y && \
+    apt-get install -y git
+
+RUN mkdir "/backup/backup"
+RUN mkdir "/backup/metrics"
+
+RUN pip install backup-github-org==1.0.4
+
+ENTRYPOINT backup-github --all -t $ACCESS_TOKEN -o /backup/backup --metrics_path /backup/metrics/${ORGANIZATION}.prom $ORGANIZATION
```

Comment:
In short: do not use entrypoint, use cmd. If you want to know more, google it - 'entrypoint vs cmd'.

If speak roughly - you don't need to care about how to run application inside container, you only need to pack it into image